js/app.js

```diff
@@ -1,205 +0,0 @@
-
-$(document).ready(function () {
-  if (!storageStatus('localStorage'))
-    $('.storageErrorStatus')[0].style.visibility = 'visible';
-  initialState();
-  var shopLists = getLists();
-  var listItems = getListsItems(shopLists[0]);
-  var checked = getListsItems(shopLists[0].concat('.Checked'));
-  parseList(shopLists, addListName);
-  parseList(listItems, addItem);
-  parseChecked(checked);
-  $('.listNames li').each(function() {
-    $(this).removeClass('highlight');
-  });
-  $('.listNames ul').find('input').first().prop('checked', true);
-  $('.listNames ul').find('li').first().toggleClass('highlight', this.checked);
-  $(document).on('mousedown', '.delete', (function(){
-    deleteList.call(this);
-  }));
-  $('#listNameForm').submit(function(e) {
-    var name = $('#listName').val();
-    if (name === '') {
-      $('#listName').addClass('error');
-    } else {
-      $('#listName').removeClass('error');
-      addListName(name);
-      var shopLists = getLists();
-      shopLists.push(name);
-      setLists('Shopping Lists', shopLists);
-      var emptyList = [''];
-      setLists(name.concat('.Checked'), emptyList);
-      $('.listItems ul').empty();
-    }
-    e.preventDefault();
```

Comment:
It's common to put the prevent default first. If anything errors through this function, the prevent default may not run.